### PR TITLE
Finally commit a demo example that works :D

### LINKED ISSUE #89: Create integration test demonstrating milestone 1
## Expected Behavior
To make sure we work in the same direction was we work towards [our first milestone](https://github.com/knative/build-pipeline/milestone/1) we should have an integration test (which would be skipped) that attempts to perform the actions we want to demo, which we can review as a team and make sure we are happy with.

## Actual Behavior
The only description of the expected behaviour is the description of milestone 1, but we are all on slightly different pages about what we need to accomplish.

## Additional Info
https://github.com/knative/build-pipeline/milestone/1

### PR DESCRIPTION
**Pull request by bobcatfish**
This example will build two images and then deploy them. It uses the
microservice example in Skaffold (just an arbitrary choice of a simple
deployment situation with 2 apps).

This is the (updated version of) the demo that @dlorenc gave at Jenkins
World.

It's kind of weird that this adds yet another example; I suggest that as
part of #108 we remove the other examples since we've never actually run
them successfully: they were added at the beginning of the project as
examples of where we wanted to go.

Had to update step names b/c using camelcase in container names is not
allowed (opened #273 to handle this).

Fixes #89

### CODE REVIEW

**Comment by bobcatfish** [comment]
> can we as part of this PR remove the kritis and guestbook files and use this demo as our example? Existing examples don't really work. I tried writing a test for them as part of #108 and it wasn't going very well

Sure let's do it!!

**Comment by bobcatfish** [comment]
Removed the examples that didn't work, made sure that the output example works.

Ready for another look @pivotal-nader-ziada 
(And @shashwathi  you might be interested in the changes to the output example)

File: examples/smoke-test.sh [hunk 1]
```diff
@@ -25,11 +25,9 @@ set -o errexit
 set -o pipefail
 
 kubectl apply -f examples/
-kubectl apply -f examples/pipelines
-kubectl apply -f examples/invocations
+kubectl apply -f examples/runs
```
Comment (shashwathi) [inline examples/smoke-test.sh]: This is a typo with `run`
I think the command is this the one below. 
```
kubectl apply -f examples/run/
```

File: examples/smoke-test.sh [hunk 2]
```diff
@@ -25,11 +25,9 @@ set -o errexit
...
 
 sleep 5
 
-kubectl delete -f examples/invocations
-kubectl delete -f examples/pipelines
+kubectl delete -f examples/runs
```
Comment (shashwathi) [inline examples/smoke-test.sh]: same here.

File: examples/smoke-test.sh (hunk 1 above)
Comment (bobcatfish) [reply examples/smoke-test.sh]: whoops! thanks @shashwathi :sweat_smile:

**Pull request merged**